### PR TITLE
KAA-399: Add 'Cell monitor' android demo application for data collection API

### PR DESCRIPTION
**Pull request by ikulikov**

### CODE REVIEW

**Comment by ashvayka** [comment]
LGTM

File: examples/cellmonitor/source/src/org/kaaproject/kaa/demo/cellmonitor/CellMonitorApplication.java [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+            /*
+             * Create instance of cell monitor log record and populate with latest values
+             */
+            CellMonitorLog cellMonitorLog = new CellMonitorLog();
+            cellMonitorLog.setLogTime(mLastLogTime);
+            cellMonitorLog.setNetworkOperatorCode(Integer.valueOf(mTelephonyManager.getNetworkOperator()));
+            cellMonitorLog.setNetworkOperatorName(mTelephonyManager.getNetworkOperatorName());
+            
+            int cid = -1;
+            int lac = -1;
```
Comment (shpigor) [inline examples/cellmonitor/source/src/org/kaaproject/kaa/demo/cellmonitor/CellMonitorApplication.java]: Move to constant

File: examples/cellmonitor/source/src/org/kaaproject/kaa/demo/cellmonitor/CellMonitorApplication.java [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+            if (mCellLocation != null && mCellLocation instanceof GsmCellLocation) {
+                GsmCellLocation gsmCellLocation = (GsmCellLocation)mCellLocation;
+                cid = gsmCellLocation.getCid();
+                lac = gsmCellLocation.getLac();
+            }
+            
+            cellMonitorLog.setGsmCellId(cid);
+            cellMonitorLog.setGsmLac(lac);
+            
+            int gsmSignalStrength = -1;
```
Comment (shpigor) [inline examples/cellmonitor/source/src/org/kaaproject/kaa/demo/cellmonitor/CellMonitorApplication.java]: Magic number?

File: examples/cellmonitor/source/src/org/kaaproject/kaa/demo/cellmonitor/CellMonitorFragment.java [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+            cid = gsmCellLocation.getCid();
+            lac = gsmCellLocation.getLac();
+        }
+        if (cid != -1) {
+            mGsmCellIdValue.setText(""+cid);
+        } else {
+            mGsmCellIdValue.setText(R.string.unavailable);
+        }
+        if (lac != -1) {
+            mGsmLacValue.setText(""+lac);
```
Comment (shpigor) [inline examples/cellmonitor/source/src/org/kaaproject/kaa/demo/cellmonitor/CellMonitorFragment.java]: Better to use String.valueOf()

**Comment by shpigor** [comment]
Reviewed

**Pull request merged**